application-boundary.tf

```diff
@@ -63,25 +63,27 @@ resource "cloudfoundry_service_instance" "k8s_cluster" {
 #   ]
 # }
 
-data "cloudfoundry_org" "gsa" {
-  name = var.broker_space.org
+# For now we are using a hand-provisioned user-provided service, not managed by Terraform
+data "cloudfoundry_space" "broker-space" {
+  name     = var.broker_space.space
+  org_name = var.broker_space.org
 }
-
-data "cloudfoundry_space" "dev-ssb" {
-  name = var.broker_space.space
-  org  = data.cloudfoundry_org.gsa.id
+data "cloudfoundry_user_provided_service" "ssb-solrcloud-k8s" {
+  name  = "ssb-solrcloud-k8s"
+  space = data.cloudfoundry_space.broker-space.id
 }
 
-# module "broker_solrcloud" {
-#   source = "./broker"
+module "broker_solrcloud" {
+  source = "./broker"
 
-#   name          = "ssb-solrcloud"
-#   path          = "./app-solrcloud"
-#   broker_space  = var.broker_space
-#   client_spaces = var.client_spaces
-#   enable_ssh    = var.enable_ssh
-#   services      = [cloudfoundry_service_instance.solrcloud_broker_k8s_cluster.id]
-# }
+  name          = "ssb-solrcloud"
+  path          = "./app-solrcloud"
+  broker_space  = var.broker_space
+  client_spaces = var.client_spaces
+  enable_ssh    = var.enable_ssh
+  # services      = [cloudfoundry_service_instance.solrcloud_broker_k8s_cluster.id]
+  services = [data.cloudfoundry_user_provided_service.ssb-solrcloud-k8s.id]
+}
```

Comment:
The Solrcloud Broker is available again?

Reply:
Yes, using the hand-propped EKS.